sls/Dockerfile

```diff
@@ -2,28 +2,36 @@ FROM alpine:3.9
 
 ENV SERVERLESS_VERSION=1.50.0 \
     NODE_VERSION=10.14.2-r0 \
-    PYTHON3_VERSION=3.6.8-r2 \
     CLOUD_SDK_VERSION=263.0.0 \
     VAULT_VERSION=1.2.0 \
     AWS_CLI_VERSION=1.16.241 \
     CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true \
-    PATH=$PATH:/usr/local/gcloud/google-cloud-sdk/bin:/usr/local/vault
+    PATH=$PATH:/usr/local/gcloud/google-cloud-sdk/bin:/usr/local/vault:/root/.pyenv/bin:/root/.pyenv/shims
 
 # Install dependencies and cleanup
-RUN apk add --update unzip jq nodejs=$NODE_VERSION nodejs-npm=$NODE_VERSION python3=$PYTHON3_VERSION yarn \
+RUN apk add --no-cache --virtual=.build-deps curl git openssl-dev bzip2-dev zlib-dev readline-dev sqlite-dev \
+  && apk add --no-cache --virutal=.run-deps bash build-base openssl readline libffi libbz2 libffi-dev sqlite sqlite-libs bzip2 zlib unzip jq nodejs=$NODE_VERSION nodejs-npm=$NODE_VERSION yarn \
+  && curl --location https://raw.githubusercontent.com/pyenv/pyenv-installer/master/bin/pyenv-installer | bash \
+  && pyenv install 3.6.8 \
+  && pyenv install 3.7.4 \
+  && pyenv global 3.6.8 3.7.4 \
+  && pyenv rehash \
+  && alias python3=python3.6 \
   && pip3 install --upgrade pip \
   && pip3 install --upgrade awscli==$AWS_CLI_VERSION \
   && mkdir ~/.aws \
-  && rm -rf /var/cache/apk/* \
   && wget -qO- https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz > /tmp/google-cloud-sdk.tar.gz \
   && mkdir -p /usr/local/gcloud \
   && tar -xf /tmp/google-cloud-sdk.tar.gz -C /usr/local/gcloud \
   && /usr/local/gcloud/google-cloud-sdk/install.sh \
-  && rm /tmp/google-cloud-sdk.tar.gz \
```

Comment:
This should still be done :) Even if it's on `/tmp` you don't want it on the final image

Reply:
I have replaced `&& rm /tmp/google-cloud-sdk.tar.gz \` and `&& rm /tmp/vault-linux-amd64.zip` with single line `&& rm -rf /tmp/*`. But good point anyay, because I forgot to add `-rf` flag (fixed now already).

Reply:
Good!